### ISSUE TITLE
[Multi-View] Consider suggesting people to color axis title for dual axis chart 

### ISSUE THREAD

**Issue by kanitw** [uwdata/visualization-curriculum#5]
like in https://vega.github.io/vega-lite/examples/layer_dual_axis.html

![image](https://user-images.githubusercontent.com/111269/57028881-18475b80-6bf5-11e9-887f-3f54d03f3af8.png)

The current example in the notebook doesn't make it clear which variable is which mark

![image](https://user-images.githubusercontent.com/111269/57028952-4d53ae00-6bf5-11e9-9a65-b07db68c2d36.png)

(It's ok for people who follow the tutorial as they created it themselves, but once  they copy this example to create charts and share with other audience, it would be confusing.)

**Comment by RandomFractals** [comment]
tbh, I still find these double Y axis graphs confusing and find them bad data viz practice. github Visitors tab is a prime example of it, but I second @kanitw suggestion to color code them if you must.

**Comment by domoritz** [comment]
Yep, they are generally considered an anti-pattern (and @kanitw is well aware of that) but there are some useful cases and you definitely need to make sure there are hints for what the lines/bars mean by coloring them according to the titles.

**Comment by RandomFractals** [comment]
agreed! good catch @kanitw!
